ingestion/functions/retrieval/retrieval_test.py

```diff
@@ -114,6 +114,14 @@ def test_format_url(mock_today):
     assert retrieval.format_source_url(
         url) == "http://foo.bar/2020-06-08/6/8.json"
 
+@patch('retrieval.retrieval.get_today')
```

Comment:
you can also use `freezegun` for this https://github.com/spulec/freezegun

Reply:
Nice! didn't know about it :)